data/crac-io/crac-io-cne/src/main/java/com/farao_community/farao/data/crac_io_cne/Cne.java

```diff
@@ -94,11 +98,20 @@ private void createAllConstraintSeries(Point point) {
         Crac crac = cneHelper.getCrac();
 
         List<ConstraintSeries> constraintSeriesList = new ArrayList<>();
-        crac.getBranchCnecs().forEach(cnec -> createConstraintSeriesOfACnec(cnec, cneHelper, constraintSeriesList));
+        List<BranchCnec> sortedCnecs = new ArrayList<>(crac.getBranchCnecs());
+        sortedCnecs.sort(Comparator.comparing(Identifiable::getId));
+        sortedCnecs.forEach(cnec -> createConstraintSeriesOfACnec(cnec, cneHelper, constraintSeriesList));
```

Comment:
I think this is more compact `personList.stream().sorted(Comparator.comparing(Person::getPersonId)).forEach(person -> System.out.println(person.getName()));`